docs/proposals/systemd-nodespec.md

```diff
@@ -0,0 +1,134 @@
+# Kubernetes Node Specification - systemd
+
+**Author**: Derek Carr (@derekwaynecarr)
+
+**Status**: Proposed
+
+*This document presents a specification for how the ```kubelet``` interfaces with ```systemd``` with a focus on quality of service concerns.*
+
+## Motivation
+
+Linux distributions have either adopted, or plan to adopt ```systemd``` as their init system.
+
+This document describes a series of enhancements that should be made to the ```kubelet``` to better integrate with these distributions.
+
+It has a primary focus on describing how the ```kubelet``` interaction with ```cgroups``` should change to work through ```systemd```.
+
+## systemd units
+
+```systemd``` manages objects referred to as ```units```.
+
+To help understand this proposal, we first define the types of ```units```:
+
+* ```service``` - a file that describes how to manage a service or application on the server; how to start/stop a service; when it should
+be started; under what circumstances it should be restarted; and any resource controls that should be applied to the service.
+* ```slice``` - a file that describes the inner-leaves of the cgroup hierarchy.  the name aligns with its placement in the hierarchy tree;
+for example, ```kubelet-besteffort.slice``` denotes a cgroup node ```/sys/fs/cgroup/<controller>/kubelet.slice/besteffort.slice```.  A
```

Comment:
It actually denotes a cgroup node `/sys/fs/cgroup/<controller>/kubelet.slice/kubelet-besteffort.slice`. The `kubelet` part is repeated in the cgroup path.